conf/chest.yaml

```diff
@@ -1 +1,2 @@
-{options: -r -LS -M 128, path: WinChest.exe}
+options: -r -LS -M 128
+path: WinChest.exe
```

Comment:
YAML reformatted, please revert, this will only produce noise in the source log.

Reply:
solved